File: projects/dotnet/crud/parkinglot/Vehicle.cs
namespace parkinglot
{
    public abstract class Vehicle
    {
        public string vehicleName { get; set; }
        public VehicleType vehicleType { get; set; }
        public Vehicle(string vehicleName, VehicleType vehicleType)
        {
            this.vehicleName = vehicleName;
            this.vehicleType = vehicleType;
        }
    }
}

File: LLDQuestions/parking lot/parking lot/src/Entities/Vehicle.java
package Entities;

public class Vehicle {
    String vehicleNumber;
    String OwnerName;
    VehicleType type;
    VehicleSize size;

    public Vehicle(String vehicleNumber, String ownerName, VehicleType type, VehicleSize size) {
        this.vehicleNumber = vehicleNumber;
        OwnerName = ownerName;
        this.type = type;
        this.size = size;
    }
}
